Server/MirObjects/MapObject.cs

```diff
@@ -63,6 +63,8 @@ public byte PercentHealth
         public int CurrentHandWeight,
                    CurrentWearWeight,
                    CurrentBagWeight;
+  
+        public byte  MoisonResist;
```

Comment:
Instead of hard coded variables - please use the new stat system.

Stats[Stat.PoisonResist]